programs/mango-v4/tests/cases/test_liq_tokens.rs

```diff
@@ -474,12 +474,10 @@ async fn test_liq_tokens_with_token() -> Result<(), TransportError> {
     .await
     .unwrap();
 
-    // Liqee's remaining collateral got dusted, only borrows remain
-    // but the borrow amount is so tiny, that being_liquidated is already switched off
     let liqee = get_mango_account(solana, account).await;
     assert_eq!(liqee.active_token_positions().count(), 1);
-    assert!(account_position_f64(solana, account, borrow_token1.bank).await > -1.0);
-    assert!(!liqee.being_liquidated());
+    assert!(account_position_f64(solana, account, borrow_token1.bank).await > -2.74);
+    assert!(liqee.being_liquidated());
```

Comment:
checking that being_liquidated turns off in the end is part of the test, I'll see if I can adjust this

Reply:
On reading the code: it's fine, bankruptcy is handled in another test. I just re-added the comment